docs/static_site/src/pages/api/api.html

```diff
@@ -123,26 +74,11 @@ <h4>Python-first API</h4>
 {%- endfor -%}
 <h2>Other Bindings</h2>
 <div class="row">
-{%- for doc in page.docs -%}
-  {%- if doc.tag != 'python' -%}
-  <div class="col-4">
-      <div class="docs-card">
-          <div class="docs-logo-container">
-              <img class="docs-logo-image" src="{{doc.icon | relative_url}}">
-          </div>
-          <div class="docs-action-btn">
-            <a href="{{doc.guide_link | relative_url}}"> <img src="{{'assets/img/compass.svg' | relative_url}}" class="docs-logo-docs">{{doc.title}} Guide  <span class="span-accented">›</span></a>
-          </div>
-          <div class="docs-action-btn">
-            <a href="{{doc.tutorial_link | relative_url}}"> <img src="{{'assets/img/video-tutorial.svg' | relative_url}}" class="docs-logo-docs">{{doc.title}} Tutorials  <span class="span-accented">›</span></a>
-          </div>
-          <div class="docs-action-btn">
-            <a href="{{doc.api_link | relative_url}}"> <img src="{{'assets/img/api.svg' | relative_url}}" class="docs-logo-docs">{{doc.title}} API Reference  <span class="span-accented">›</span></a>
-          </div>
-      </div>
+  <div class="language-binding-banner">
+    <h4>Call for Contribution</h4>
+    The Clojure, Java, Julia, R, and Scala language bindings of MXNet v1.x were removed in v2.x due to some C APIs being deprecated and the bindings rely on the deprecated APIs. You can still use these language bindings in v1.x.
```

Comment:
We should create an issue, if not already exists, with list of C APIs deprecated.

Reply:
and refer it here.

Reply:
Done, added the reference github issue: https://github.com/apache/incubator-mxnet/issues/17676